server/src/test/java/io/spine/examples/chatspn/server/chat/given/ChatTestEnv.java

```diff
@@ -75,7 +73,9 @@ public static CreatePersonalChat createPersonalChatCommand() {
                 .newBuilder()
                 .setId(ChatId.generate())
                 .setCreator(GivenUserId.generated())
+                .setCreatorName(randomString())
                 .setMember(GivenUserId.generated())
+                .setMemberName(randomString())
```

Comment:
Please consider generating some real-looking names instead of just random strings, in tests. It is generally a good idea to follow a common format (in this case, "Vasya Pupkin", being the first name and the last name joined with a space).